composer.json

```diff
@@ -20,6 +20,11 @@
             "services": {
                 "drush.services.yml": "^9"
             }
+        },
+        "patches": {
+            "drupal/commerce_shipping": {
+                "Add Shipments tab to Order page": "https://www.drupal.org/files/issues/2018-06-27/commerce_shipping-shipments-tab-2981945-2.patch"
+            }
```

Comment:
Did this actually work? It's good as documentation anyway.

Reply:
dont know yet, cuz i tried to run `composer update` and nothing happened. but i figured once these changes are committed to the branch, composer might pick them up

Reply:
Ok, I don't think it will be picked up, but even if it doesn't let's keep it so that it's documented. Required patches could also go to the README or other doc file.